test/integration/fixtures/ms-word-out.html

```diff
@@ -37,3 +37,43 @@ <h2>This is a heading level 2</h2>
 <!-- wp:image -->
 <figure class="wp-block-image"><img src="" alt=""/></figure>
 <!-- /wp:image -->
+
+<!-- wp:paragraph -->
+<p><a href="#anchor">This is an anchor link</a> that leads to the next paragraph.</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p><a id="anchor">This is the paragraph with the anchor.</a></p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p><a href="#nowhere">This is an anchor link</a> that leads nowhere.</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p><a>This is a paragraph with an anchor with no link pointing to it.</a></p>
```

Comment:
@ellatrix Could you provide some more guidance on how could we unwrap text within links with no attributes (therefore with no use)? You mentioned removeInvalidHTML should handle it but I couldn't figure out how we could make sure it achieves that (it doesn't at the moment). It would be nice to clean it up a bit.